ONSdigital/census-rm-load-generator
# Load generator env terraform

# Motivation and Context

We want to have an environment separate from the wargames env that provides load so that the traffic isn't coming from within so to speak.
# What has changed



Terraform files and apply script and associated files added.
# How to test?



Look at the rm loadgen env on GCP and check it out, alternatively build your own by following the instructions in the readme within the loadgen-terraform folder.
# Links



https://trello.com/c/ChYRVvPr/1193-environment-to-provide-load-generation-for-wargaming-capacity-testing-8
